eventuate-cdc-data-producer-wrappers/src/main/java/io/eventuate/cdc/producer/wrappers/kafka/TopicPartitionSender.java

```diff
@@ -97,10 +74,41 @@ But other thread added message to queue (just after we got null message) and tri
     }
   }
 
+
+  private byte[] processBatchIfNecessary(TopicPartitionMessage message) {
+    if (!enableBatchProcessing) {
+      return message.getBody().getBytes(Charset.forName("UTF-8"));
```

Comment:
I'm guessing that string.getBytes() and new String(...) is duplicated in many places in the code.
The encoding of message bodies should be defined in one place, ie. class with fromBytes/toBytes static methods

Reply:
ok